doc/Learn/Navigation/HowTo-Regions.md

```diff
@@ -0,0 +1,135 @@
+---
+uid: Uno.Extensions.Navigation.Regions
+---
+# How-To: Define Regions
+
+A Region is used to link specific sectors of a view to individual items on a navigation control within the same page. This helps in managing and organizing navigation within different sections of a view, ensuring that each section can independently display and navigate through content. Regions are usually used to manage the navigation of pages and to display or hide content with controls like `NavigationView` and `TabBars`.
+
+> [!IMPORTANT]
+> When working with `NavigationView` or `TabBar`, it's crucial not to define them within `ShellView`. Instead, these controls should be defined on your `MainPage` or another appropriate page in your application.
+>
+> [!NOTE]
+> If an app only uses basic navigation between pages without any nested views or pages, it is not necessary to register routes. In such cases, the default `Frame` and navigator are sufficient to manage the navigation, simplifying the setup and reducing the need for additional configuration.
+
+## Properties in the Region Class
+
+1. **`Region.Attached`**:
+   - **Description**: This property is used to mark a sector of the view as a region. It links the navigational control with the container provided to render content.
+   - **Example**: `<Grid uen:Region.Attached="True">`
+
+1. **`Region.Navigator`**:
+   - **Description**: Defines the navigation behavior for the region, determining how the region responds to navigation changes.
+   - **Values**: `Visibility`.
```

Comment:
Is Visibility the only named navigator that we can put here? We can't force other things like a FrameNavigator?